### PR TITLE
Refactor/modelica builder

### PR DESCRIPTION
**Pull request by macintoshpie**
#### Any background context you want to provide?
Moving toward using the modelica-builder package
Unfortunately this is slower at processing modelica files. After some investigation it's mostly due to parsing the modelica file (ie out of our control).
#### What does this PR accomplish?
Integrates the modelica-builder in teaser.py
#### What are the relevant tickets?
#76 

Note: I 'tested' it by diffing the files generated by the tests since the tests themselves don't seem to check the output. @nllong please let me know if this is insufficient

### CODE REVIEW

**Comment by nllong** [comment]
If the unit tests pass, then whatever you did definitely works! I'll review shortly.

File: geojson_modelica_translator/model_connectors/teaser.py [hunk 1]
```diff
@@ -178,71 +179,98 @@ def post_process(self, scaffold, building_names, keep_original_models=False):
...
 
                 # add new port connections
                 if self.system_parameters.get_param(
                         "buildings.default.load_model_parameters.rc.order", default=2) == 1:
-                    data = "annotation (Line(points={{0,100},{96,100},{96,20},{92,20}}, color={191,0,0}))"
-                    mofile.add_connect("port_a", "thermalZoneOneElement.intGainsConv", data)
-
-                    data = "annotation (Line(points={{93,32},{98,32},{98,0},{110,0}}, color={0,0,127}))"
-                    mofile.add_connect("thermalZoneOneElement.TAir", "TAir", data)
+                    mofile.add_connect(
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/teaser.py:237]: this looks very nice!

**Comment by nllong** [comment]
So I think we should merge this down but keep the old input parser in there until we see the ripple effect of the performance. thoughts?

**Comment by nllong** [comment]
@macintoshpie  -- can you resolve this conflict? There was a new component that we added (fraction latent) that needs to be added to the new modelica parser. Thanks!

**Comment by macintoshpie** [comment]
This PR has been moved to #102 to address some changes

**Pull request closed without merging**